controllers/idler/idler_controller.go

```diff
@@ -141,6 +155,91 @@ func (r *Reconciler) ensureIdling(logger logr.Logger, idler *toolchainv1alpha1.I
 	return r.updateStatusPods(idler, newStatusPods)
 }
 
+func (r *Reconciler) createNotification(logger logr.Logger, idler *toolchainv1alpha1.Idler) error {
+	logger.Info("Create Notification")
+	//Get the HostClient
+	hostCluster, ok := r.GetHostCluster()
+	if !ok {
+		return fmt.Errorf("unable to get the host cluster")
+	}
+	//check the condition on Idler if notification already sent, only create a notification if not created before
+	_, found := condition.FindConditionByType(idler.Status.Conditions, toolchainv1alpha1.IdlerTriggeredNotificationCreated)
+	if found && condition.IsTrue(idler.Status.Conditions, toolchainv1alpha1.IdlerTriggeredNotificationCreated) {
```

Comment:
As far as I remember `IsTrue()` returns false if the condition is not found.
```suggestion
	if condition.IsTrue(idler.Status.Conditions, toolchainv1alpha1.IdlerTriggeredNotificationCreated) {
```